runtimes/php/tutorial-laravel.md

```diff
@@ -98,3 +98,30 @@ Create a new environment variable called `CC_FS_BUCKET` and set `/storage/app:<b
 If you need to build your frontend assets (eg. javascript or CSS files), you can either add it as a step in your composer file, or you can add a post build hook with the `CC_POST_BUILD_HOOK` environment variable.
 
 For example, if you launch the build with `npm run prod`: `CC_POST_BUILD_HOOK=npm install && npm run prod`.
+
+### Optional: configure task scheduling
+
+If your app uses [task scheduling](https://laravel.com/docs/scheduling), you need to configure a cron:
+
+1. Create a `clevercloud/cron.json` file in your project, containing:
+
+```json
+[
+    "* * * * * $ROOT/clevercloud/cron.sh"
+]
+```
+
+2. Create a `clevercloud/cron.sh` file in your project (with execute permissions), containing:
+
+```bash
+#!/bin/bash -l
+set -euo pipefail
+
+cd "$APP_HOME" || exit 1
```

Comment:
```suggestion
pushd "$APP_HOME"
```

`pushd` will fail if the directory doesn't exists and since you already have the `set -e`, the entire script will fail

Reply:
Thanks I didn't know `pushd`!